describe.py

```diff
@@ -435,7 +433,7 @@ def document_api(name, version, uri, doc_destination_dir):
         return
 
     document_collection_recursive(
-        service, "{}_{}.".format(name, version), discovery, discovery, doc_destination_dir
+        service, "{}_{}.".format(name, safe_version(version)), discovery, discovery, doc_destination_dir
     )
```

Comment:
I think this change also needs to be made in `document_api_from_discovery_document`.

Reply:
No change needed in `document_api_from_discovery_document`, as it's already being done here. https://github.com/googleapis/google-api-python-client/blob/master/describe.py#L457